src/current/_includes/common/basic-terms.md

```diff
@@ -36,7 +36,7 @@ However, 'legacy service accounts' that were created before the updated authoriz
 
 To learn more, refer to [Manage Service Accounts]({{ link_prefix }})managing-access.html#manage-service-accounts)
 
-#### CockroachDB {{ site.data.products.serverless }} cluster
+#### CockroachDB {{ site.data.products.basic }} cluster
 
 A CockroachDB {{ site.data.products.cloud }} cluster deployed on request for a specific customer in *shared* network and compute infrastructure.
```

Comment:
Is this the right terminology? Might be an opportunity to say "A CockroachDB cluster deployed on {{ site.data.products.cloud }} for a specific customer...." I don't think we want to say the type of cluster is a Cloud one -- it's actually a Basic one.

Reply:
Nah it can be either standard or basic (+ not focusing on existing content in this PR)